source/metadata_management_heritage.rst

```diff
@@ -6,8 +6,101 @@ MET Norway
 
 Service catalog
 ^^^^^^^^^^^^^^^
+The software library used to implement a Service Catalogue is based on the PyCSW python library.
+pycsw enables users to publish a catalogue of dataset descriptions (metadata) to the web, using standards based interfaces and formats, such that it can be queried and updated by catalogue clients. pycsw is simple to setup within an Apache web server (or any WSGI server) and can access metadata records stored as XML files within a database.
+
+Metadata is 'data about data', storing such things as creation-date, author, title, area-of-interest, etc, and is usually encoded as XML files, following international standards such as ISO 19115, ISO 19139 and FGDC CSDGM.
+PyCSW is the software library used to provide a catalogue service. PyCSW implements the  `OGC Catalogue Services for the Web (CSW) standard <https://www.opengeospatial.org/standards/cat>`__. The project is certified `OGC Compliant`_, and is an OGC Reference Implementation. The CSW standard defines common interfaces to discover, browse and query metadata about spatial data, web services and related resources.
+
+Core Features
```

Comment:
Could you write out the acronyms and link to the resources as well? That could make it easier to understand for a non-expert :)

Reply:
I expanded most of the acronyms and added some links to the metadata specification of the several standards mentioned.